README.md

```diff
@@ -437,7 +438,13 @@ execute those requests. The backend should not allow the scheduler
 thread to return from TRITONBACKEND_ModelInstanceExecute until that
 instance is ready to handle another set of requests. Typically this
 means that the TRITONBACKEND_ModelInstanceExecute function will
-create responses and release the requests before returning.
+create responses and release the requests before returning. However,
+in case TRITONBACKEND_ModelInstanceExecute returns an error, the ownership
+of requests is transferred back to Triton which will then be responsible
+for releasing them. Backend must not retain refrences to the requests
```

Comment:
"... for releasing them. Therefore, in the case where TRITONBACKEND_ModelInstanceExecute returns an error, the backend must not retain... "